src/components/Offers/New/CreateOfferForm.js

```diff
@@ -0,0 +1,475 @@
+                                                </Grid>
+
+                                                <Grid item xs={12} lg={6} className={classes.gridWithInfo}>
+                                                    <IsHiddenComponent
+                                                        disabled={disabled}
+                                                        errors={errors}
+                                                        requestErrors={requestErrors}
+                                                        control={control}
+                                                    />
+                                                </Grid>
+                                            </Grid>
+
+                                        </Collapse>
+                                    </Grid>
+                                </Grid>
+
+                                <MultiOptionTextField
+                                    values={contacts}
+                                    label="Contacts"
+                                    itemLabel="Contact #"
+                                    controllerName="contacts"
+                                    onAdd={appendContact}
+                                    onRemove={removeContact}
+                                    getValues={getValues}
+                                    control={control}
+                                    errors={errors.contacts || requestErrors.contacts}
+                                    disabled={disabled}
+                                    addEntryBtnTestId="contacts-selector"
+                                />
+                                <MultiOptionTextField
+                                    values={requirements}
+                                    label="Requirements"
+                                    itemLabel="Requirement #"
+                                    controllerName="requirements"
+                                    onAdd={appendRequirement}
+                                    onRemove={removeRequirement}
+                                    getValues={getValues}
+                                    control={control}
+                                    errors={errors.requirements || requestErrors.requirements}
+                                    disabled={disabled}
+                                    textFieldProps={{ multiline: true }}
+                                    addEntryBtnTestId="requirements-selector"
+                                />
+
+                                <TextEditorComponent
+                                    fields={fields}
+                                    disabled={disabled}
+                                    errors={errors}
+                                    requestErrors={requestErrors}
+                                    control={control}
+                                />
+
+                                <Grid item xs={12} lg={12} />
```

Comment:
This is doing nothing. Did you mean to wrap the components below, or maybe remove?

Also, I noticed the components for contacts, requirements and so on are not in a Grid container. Doing so would help the mobile design. I think just having a container with an item xs=12 would not change the appearance now and would help the design in mobile